### PR TITLE
Created mihai.json!

### PR DESCRIPTION
**Pull request by 8xu**

### CODE REVIEW

**Comment by 8xu** [comment]
Is there anyone here? 😸

**Review by phenax — DISMISSED** [review]
Sorry for the delayed response. The json file needs to contain a single record. You can look at existing records for reference.

**Comment by 8xu** [comment]
> Sorry for the delayed response. The json file needs to contain a single record. You can look at existing records for reference.

Can I link it on Cloudflare?

**Comment by phenax** [comment]
I'm unfamiliar with the setup process but I think it should be possible

**Comment by 8xu** [comment]
Fixed it, should be good now to be honest! :)

File: domains/mihai.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "repo": "https://github.com/8xu/web",
+  "owner": {
+    "username": "8xu",
+    "email": "mihai@8xu.tech",
+    "twitter": "reallymihai"
+  },
+  "record": {
+    "CNAME": "https://8xu.github.io/web/"
```
Comment (TheBotlyNoob) [inline domains/mihai.json:9]: a `CNAME` cannot have a `/`. You can rename the repository to `8xu.github.io`, and then change the `CNAME` to `8xu.github.io`, or if you really want to keep that repository, you can change it to a `URL` record.

**Review by TheBotlyNoob — DISMISSED** [review]

**Review by MaskDuck — APPROVED** [review]

**Comment by MaskDuck** [comment]
@8xu sorry for the delay but your domain is now accepted!

**Comment by 8xu** [comment]
Thanks!

**Pull request merged**